package.json

```diff
@@ -792,6 +792,16 @@
                                 "description": "Enables automatic population of the debug variable panel on a breakpoint or runtime errors.",
                                 "default": true
                             },
+                            "autoResolveVirtualVariables": {
+                                "type": "boolean",
+                                "description": "(Experimental) Enables automatic population of the virtual variables.",
+                                "default": false
+                            },
+                            "enhanceREPLCompletions": {
+                                "type": "boolean",
+                                "description": "Enables scanning deployment files for additional REPL completions,\nsuch as user-defined functions. This process runs in a background\nthread and may be resource-intensive.",
```

Comment:
why are we putting newlines in this? I don't think they're necessary in this context.